packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -556,12 +560,16 @@ class PageView extends StatefulWidget {
 }
 
 class _PageViewState extends State<PageView> {
-  int _lastReportedPage = 0;
+  int _onPageChangedIndex = 0;
```

Comment:
Both of these names should probably still indicate that it is the last reported index.